### PR TITLE
Use correct object for session

### PR DESCRIPTION
**Pull request by vrutkovs**
PR #38 used an incorrect variable for session and NVR check was not performed

### CODE REVIEW

File: koji_containerbuild/plugins/builder_containerbuild.py [hunk 1]
```diff
@@ -646,8 +646,9 @@ def handler(self, src, target, opts=None):
             if not self.opts.get('scratch'):
                 expected_nvr = labels_wrapper.get_expected_nvr()
                 try:
-                    build_id = self.xmlrpc.getBuild(expected_nvr)['id']
+                    build_id = self.session.getBuild(expected_nvr)['id']
                 except:
+                    self.logger.info(''.join(traceback.format_exception(*sys.exc_info())))
```
Comment (twaugh) [inline koji_containerbuild/plugins/builder_containerbuild.py]: Could just add `exc_info=True` to the existing self.logger.info call.

Comment (vrutkovs) [inline koji_containerbuild/plugins/builder_containerbuild.py]: Right, updating

**Comment by lcarva** [comment]
LGTM

**Comment by breillyrh** [comment]
LGTM too, I'll merge.

**Pull request merged**